stage1/enter_kvm/enter_kvm.go

```diff
@@ -38,6 +40,92 @@ var (
 	appName string
 )
 
+// fileAccessible checks if the given path exists and is a regular file
+func fileAccessible(path string) bool {
+	if info, err := os.Stat(path); err == nil {
+		return info.Mode().IsRegular()
+	}
+	return false
+}
+
+func sshPrivateKeyPath() string {
+	return filepath.Join(kvmSettingsDir, kvmPrivateKeyFilename)
+}
```

Comment:
yes, but i need to remember to add "/" or not after each const. See there also that you cannot use filepath.Join in const area